core/src/main/java/io/fabric8/maven/core/service/openshift/OpenshiftBuildService.java

```diff
@@ -282,11 +299,123 @@ private BuildStrategy createBuildStrategy(ImageConfiguration imageConfig, OpenSh
                     .endFrom()
                     .endSourceStrategy()
                     .build();
+            if (openshiftPullSecret != null) {
```

Comment:
same comment here like `getDockerStrategy`, if applicable?

Reply:
@hrishin Thanks for your input.
Modified as per your comments & pushed.